calculator.py

```diff
@@ -1,2 +1,7 @@
 def add(x, y):
   return x+y
+def subtract(x, y):
+  return x-y
```

Comment:
can you make a variable and store the value of the subtraction to that variable then return the variable.